content/en/real_user_monitoring/error_tracking/mobile/ios.md

```diff
@@ -146,10 +146,193 @@ To enable app hang monitoring:
+                          }
+                      ],
+                      "compute": {
+                          "aggregation": "sum",
+                          "metric": "@freeze.duration",
+                          "interval": 3600000
+                      },
+                      "storage": "hot"
+                  },
+                  {
+                      "name": "query1",
+                      "data_source": "rum",
+                      "search": {
+                          "query": "@type:session @session.type:user"
+                      },
+                      "indexes": [
+                          "*"
+                      ],
+                      "group_by": [
+                          {
+                              "facet": "@application.name",
+                              "limit": 10,
+                              "sort": {
+                                  "aggregation": "sum",
+                                  "order": "desc",
+                                  "metric": "@session.time_spent"
+                              },
+                              "should_exclude_missing": true
+                          },
+                          {
+                              "facet": "version",
+                              "limit": 10,
+                              "sort": {
+                                  "aggregation": "sum",
+                                  "order": "desc",
+                                  "metric": "@session.time_spent"
+                              },
+                              "should_exclude_missing": true
+                          }
+                      ],
+                      "compute": {
+                          "aggregation": "sum",
+                          "metric": "@session.time_spent",
+                          "interval": 3600000
+                      },
+                      "storage": "hot"
+                  }
+              ],
+              "response_format": "timeseries",
+              "style": {
+                  "palette": "dog_classic",
+                  "order_by": "values",
+                  "line_type": "solid",
+                  "line_width": "normal"
+              },
+              "display_type": "line"
+          }
+      ],
+      "yaxis": {
+          "include_zero": true,
+          "scale": "sqrt"
+      },
+      "markers": [
+          {
+              "value": "y > 12000000000",
+              "display_type": "error dashed"
+          },
+          {
+              "value": "6000000000 < y < 12000000000",
+              "display_type": "warning dashed"
+          },
+          {
+              "value": "0 < y < 6000000000",
+              "display_type": "ok dashed"
+          }
+      ]
+  }
+  ```
+</details>
+
 #### Disable app hang monitoring
 
 To disable app hang monitoring, update the initialization snippet and set the `appHangThreshold` parameter to `nil`.
 
+### Add watchdog terminations reporting
```

Comment:
```suggestion
### Add Watchdog Terminations reporting
```

Reply:
We've been following a lowercase format for app hangs already, so to remain consistent I used the same format for watchdog terminations